framework/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
+- Enh: migrate command can create foreign keys.
```

Comment:
can you merge this PR and we discuss it on the next PR when its about the prefixes? I have one prepared on this branch https://github.com/yiisoft/yii2/compare/master...Faryshta:migrationPrefix?expand=1